tools/lib/generateJSONSchema.js

```diff
@@ -610,6 +610,10 @@ const generateJSONSchema = (sport /*: string*/) => {
 							gameOver: {
 								type: "boolean",
 							},
+							goatFormula: {
```

Comment:
This should just be `type: "string"` because it's not wrapped. (Checking of wrapped game attributes is not very good here, as you can see...)